app/models/gsa18f/procurement.rb

```diff
@@ -26,14 +26,14 @@ def self.purchase_amount_column_name
     validates :product_name_and_description, presence: true
     validates :recurring_interval, presence: true, if: :recurring
 
-    after_create :add_approvals, :add_observers
-
-    def add_approvals
-      self.proposal.approver = User.for_email(Gsa18f::Procurement.approver_email)
-    end
-
-    def add_observers
-      self.add_observer(Gsa18f::Procurement.purchaser_email)
+    after_create :add_steps
```

Comment:
this doesn't have to happen now, but do you have thoughts on a way we could do this without a callback? I've been working to remove as many callbacks as possible!